### PR TITLE
creating first mill

### PR DESCRIPTION
**Pull request by GorillasHabitat**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  constructor({
+    id = 'coinbase',
+    socketServerUrl = 'wss://ws-feed.exchange.coinbase.com',
+    // @ts-ignore
+    pairs = ['BTC-USDT'],
+  }) {
+    super();
+
+    if (id) this.setId(id);
+    this.pairs = pairs;
```
Comment (LionsHabitat) [inline src/index.ts]: Debating if we call these collections, I guess technically it's not origami however.

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+    if (d.type !== 'ticker') return;
+
+    const paper = {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+      data: { ...d, price: Number(d.price) },
+      created: new Date(),
+    };
+
+    this.emitter.publish(`mill.${this.id ?? 'undefined'}`, paper);
```
Comment (LionsHabitat) [inline src/index.ts]: Non blocking but you are using semi colons in this file and none in the others.

**Review by LionsHabitat — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (GorillasHabitat) [reply src/index.ts]: yeah this is internal only to the mill and should be accessible outside like this

File: src/index.ts (hunk 2 above)
Comment (GorillasHabitat) [reply src/index.ts]: copying the prettierrc

**Pull request merged**